seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumAsyncJSRunner.scala

```diff
@@ -32,12 +32,8 @@ class SeleniumAsyncJSRunner(browserProvider: SeleniumBrowser,
   }
 
   override def stop(): Unit = synchronized {
-    if (!keepAlive || ignoreKeepAlive) {
-      future.onComplete { _ =>
-        browser.processConsoleLogs(console)
-        browser.close()
-      }
-    }
+    if (browser.isOpened && !keepAlive || ignoreKeepAlive)
```

Comment:
Add parentheses around the `&&` part. Don't rely on `&&` having higher precedence than `||`.